src/pybind/mgr/dashboard/run-tox.sh

```diff
@@ -17,11 +17,14 @@ fi
 
 source ${MGR_DASHBOARD_VIRTUALENV}/bin/activate
 
+ENV_LIST="cov-init,"
+if [ "$WITH_PYTHON2" = "ON" ]; then
+  ENV_LIST+="py27,"
+fi
 if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST="cov-init,py27,py3,cov-report,lint"
-else
-  ENV_LIST="cov-init,py27,cov-report,lint"
+  ENV_LIST+="py3,"
 fi
+ENV_LIST+="cov-report,lint"
```

Comment:
So, the change here is that when WITH_PYTHON2=ON and WITH_PYTHON3=OFF, py3 is not included in ENV_LIST

That seems reasonable to me

Reply:
actually, the change is to get the test pass, even if WITH_PYTHON2=OFF and WITH_PYTHON=ON.